src/store.ts

```diff
@@ -45,13 +45,24 @@ export class Store<T> {
 
   private dispatchQueue: DispatchQueueItem<T>[] = [];
 
+  private enableDevTools: boolean = true;
+
   constructor(private initialState: T, options?: Partial<StoreOptions>) {
     this.options = options || {};
     const isUndoable = this.options.history && this.options.history.undoable === true;
     this._state = new BehaviorSubject<T>(initialState);
     this.state = this._state.asObservable();
+    
+    // Do we have options, as well as devToolsOptions and the disable option on the options object?
+    if (typeof this.options.devToolsOptions !== "undefined" && typeof this.options.devToolsOptions.disable !== "undefined") {
+      if (this.options.devToolsOptions.disable === true) {
+        this.enableDevTools = false;

```

Comment:
why would you store this to the `enableDevTools` prop? Its not used anywhere else except the next conditional. Or is there anything else planned with it?

Reply:
That was because initially, I was making it testable, but I don't think it needs to be. I've made this a variable now.